wpeverest/user-registration
# Refactor - Settings page design and codes

### All Submissions:






### Changes proposed in this Pull Request:
Previously we had the default WordPress settings page design for our plugin's design. This PR introduces a new design structure for the settings page and some refactors of the codes that are responsible for achieving the design and improve the performance of the settings page.

### How to test the changes in this Pull Request:

1. Install and activate all the addons.
2. Navigate to the User Registration -> Settings page.
3. Verify if there are no breaking changes in the design.
4. Verify that all the settings are working properly and try to change the settings to see if they have any breaking conflicts in their operating areas.

### Previous Design:
![image](https://user-images.githubusercontent.com/85762481/124417151-3c27fe00-dd78-11eb-9b92-946c08adaeab.png)

### New Design:
![image](https://user-images.githubusercontent.com/85762481/124417216-5a8df980-dd78-11eb-913f-70b7b931c7b0.png)

### Types of changes:




### Other information:




### Changelog entry

> Refactor - Settings page design and codes.
